use-timescale/migration/about-hypershift.md

```diff
@@ -67,19 +67,30 @@ URIs in the configuration file.
 To see all available configuration options for hypershift, use this command:
 
 ```bash
-docker run -ti timescale/hypershift:0.5 --help
+docker run -ti timescale/hypershift:0.6 --help
 ```
 
+### The hypershift clone tool
+
+You can use the hypershift `clone` command to preserve a copy of your database
+in the source location. With this command, instead of moving the contents off
+the source and into the target, hypershift makes a full copy of the contents of
+your source database, and moves it into the source database. You can add
+parameters to the `clone` command to filter which contents to copy.
+
 |Short command|Long command|Example use|Description|
 |-|-|-|-|
-|-c|--config-file|-C <CONFIG_FILE>|Path for hypershift configuration file|
 |-s|--source|-s <SOURCE_URI>|URI to the source database|
 |-t|--target|-t <TARGET_URI>|URI to the target database|
 ||--source-password|--source-password|Password for the source database|
 ||--target-password|--target-password|Password for the target database|
-||--ignore-missing-time-index||Ignore that hypertables may be missing a time index|
+||--parallel|--parallel <NUMBER\>|Number of workers that can simultaneously fetch data from the target database|
```

Comment:
Is the `\` in `<NUMBER\>` intentional? There is also another one at 92, `<PATH\>`